src/Chunk.cpp

```diff
@@ -2278,6 +2299,38 @@ bool cChunk::DoWithBeaconAt(int a_BlockX, int a_BlockY, int a_BlockZ, cBeaconCal
 
 
 
+bool cChunk::DoWithBrewingstandAt(int a_BlockX, int a_BlockY, int a_BlockZ, cBrewingstandCallback & a_Callback)
+{
```

Comment:
I agree that refactoring the existing code is out of scope for this PR, but I think that we should use what ever new method we decide on for accessing brewing stands.